### PR TITLE
Add : Profile Screen, Profile View Model, Profile Controller

### PR DESCRIPTION
**Pull request by nikunjstackapp**
Add :
dependency : image picker, permission handler
String Constant : strings
widget : Alert dialog
utils : permission, validation

### CODE REVIEW

File: lib/app/app/utills/permission.dart
Comment (justsaagar) [inline lib/app/app/utills/permission.dart:1]: Manage it properly if any of one is denied then why you asking al two permissions? @nikunjstackapp

File: lib/main.dart
Comment (justsaagar) [inline lib/main.dart:1]: User named route for navigations, @nikunjstackapp

Comment (nikunjstackapp) [reply lib/main.dart:1]: you mean route helper files are not there?

File: lib/app/app/utills/permission.dart
Comment (nikunjstackapp) [reply lib/app/app/utills/permission.dart:1]: okay

Comment (nikunjstackapp) [reply lib/app/app/utills/permission.dart:1]: changes done

**Review by justsaagar — APPROVED** [review]

**Pull request merged**